libraries/Update/src/Update.h

```diff
@@ -22,168 +22,169 @@
-    */
-    void printError(Print &out);
-
-    const char * errorString();
-
-    /*
-      sets the expected MD5 for the firmware (hexString)
-    */
-    bool setMD5(const char * expected_md5);
-
-    /*
-      returns the MD5 String of the successfully ended firmware
-    */
-    String md5String(void){ return _md5.toString(); }
-
-    /*
-      populated the result with the md5 bytes of the successfully ended firmware
-    */
-    void md5(uint8_t * result){ return _md5.getBytes(result); }
-
-    //Helpers
-    uint8_t getError(){ return _error; }
-    void clearError(){ _error = UPDATE_ERROR_OK; }
-    bool hasError(){ return _error != UPDATE_ERROR_OK; }
-    bool isRunning(){ return _size > 0; }
-    bool isFinished(){ return _progress == _size; }
-    size_t size(){ return _size; }
-    size_t progress(){ return _progress; }
-    size_t remaining(){ return _size - _progress; }
-
-    /*
-      Template to write from objects that expose
-      available() and read(uint8_t*, size_t) methods
-      faster than the writeStream method
-      writes only what is available
-    */
-    template<typename T>
-    size_t write(T &data){
-      size_t written = 0;
-      if (hasError() || !isRunning())
-        return 0;
-
-      size_t available = data.available();
-      while(available) {
-        if(_bufferLen + available > remaining()){
-          available = remaining() - _bufferLen;
-        }
-        if(_bufferLen + available > 4096) {
-          size_t toBuff = 4096 - _bufferLen;
-          data.read(_buffer + _bufferLen, toBuff);
-          _bufferLen += toBuff;
-          if(!_writeBuffer())
-            return written;
-          written += toBuff;
-        } else {
-          data.read(_buffer + _bufferLen, available);
-          _bufferLen += available;
-          written += available;
-          if(_bufferLen == remaining()) {
-            if(!_writeBuffer()) {
-              return written;
-            }
-          }
-        }
-        if(remaining() == 0)
-          return written;
-        available = data.available();
-      }
-      return written;
-    }
-
-    /*
-      check if there is a firmware on the other OTA partition that you can bootinto
-    */
-    bool canRollBack();
-    /*
-      set the other OTA partition as bootable (reboot to enable)
-    */
-    bool rollBack();
-
-  private:
-    void _reset();
-    void _abort(uint8_t err);
-    bool _writeBuffer();
-    bool _verifyHeader(uint8_t data);
-    bool _verifyEnd();
-    bool _enablePartition(const esp_partition_t* partition);
-
-
-    uint8_t _error;
-    uint8_t *_buffer;
-    uint8_t *_skipBuffer;
-    size_t _bufferLen;
-    size_t _size;
-    THandlerFunction_Progress _progress_callback;
-    uint32_t _progress;
-    uint32_t _paroffset;
-    uint32_t _command;
-    const esp_partition_t* _partition;
-
-    String _target_md5;
-    MD5Builder _md5;
-
-    int _ledPin;
-    uint8_t _ledOn;
+typedef void (*LED_Init_t)(void);
```

Comment:
reformatting the code makes the diff bloated and a bit more challenging to know exactly what you are changing